src/components/header/Hero.js

```diff
@@ -17,42 +23,42 @@ const Hero = ({ bounceInAnimation, executeScroll }) => {
                         fontWeight: '500'
                     }}
                 >
-                    Kromatika DEX Trade
+                    KROMATIKA — next generation DEX Trading, powered by Uniswap and Chainlink
                 </h1>
                 <p
                     className="text-4xl mb-10 w-3/5 text-regular subpixel-antialiased leading-tight
 							                  leading-snug"
                 >
                     <br />
-                    Next generation DEX crypto trading:
+                    Audited smart contracts for creating trades and processing them, all powered by Uniswap
```

Comment:
somehow we lose the original information with this text changes. I prefer to have it and make it more concise since this is the front page.